docs/operator-guides/upgrading-airbyte.md

```diff
@@ -80,31 +93,28 @@ If you did not start Airbyte from the root of the Airbyte monorepo, you may run
 
 :::danger
 
-This will completely reset your Airbyte deployment back to scratch and you will lose all data.
+This will completely reset your Airbyte deployment. You will lose all data.
 
 :::
 
-## Upgrading on K8s using Helm
 
-The instructions below are for users using custom deployment and have a `values.yaml`. If you're not using a `values.yaml` to deploy Airbyte using Helm can jump directly to step `4.`.
 
-1. Access [Airbyte ArtifactHub](https://artifacthub.io/packages/helm/airbyte/airbyte) and select the version you want to upgrade.
-2. You can click in `Default Values` and compare the value file between the new version and version you're running. You can run `helm list -n <NAMESPACE>` to check the CHART version you're using.
-3. Update your `values.yaml` file if necessary.
-4. Upgrade the Helm app running:
+## Mandatory Intermediate Upgrade
 
-   ```bash
-   helm upgrade --install <RELEASE-NAME> airbyte/airbyte --values <VALUE.YAML> --version <HELM-APP-VERSION>
-   ```
+**If your current version of airbyte is < v0.32.0-alpha-patch-1, you first need to upgrade to this version before upgrading to any later version.**
```

Comment:
Version doesn't seem to render correctly in this, should it be formatted differently?

Reply:
I'm not sure. I don't see special formatting on other examples where a version is named. But maybe it's better in the update with the collapsible section.